lib/mayaUsd/render/vp2RenderDelegate/proxyRenderDelegate.h

```diff
@@ -41,9 +42,9 @@
 #endif
 
 // The new Maya point snapping support doesn't require point snapping items any more.
-#if MAYA_API_VERSION >= 20230000
-// The new Maya point snapping support has some known issues. Disable it for now.
-// #define MAYA_NEW_POINT_SNAPPING_SUPPORT
+#if (MAYA_API_VERSION >= 20220200 && MAYA_API_VERSION < 20230000) || (MAYA_API_VERSION >= 20230000 && MAYA_PREVIEW_RELEASE_VERSION >= 127)
+#define MAYA_NEW_POINT_SNAPPING_SUPPORT
+#define MAYA_SNAP_TO_SELECTED_OBJECTS_SUPPORT
```

Comment:
The expression here ended up being the same for MAYA_NEW_POINT_SNAPPING_SUPPORT and MAYA_SNAP_TO_SELECTED_OBJECTS_SUPPORT. I was thinking of removing the snap to selected one and simplifying things a bit. I don't see any downside to that, do you?

Reply:
Yes, they could be merged.

Reply:
Looks like the conditional will require an update in the future. Do you want to future-proof it with:
```
(MAYA_PREVIEW_RELEASE_VERSION >= 127 || MAYA_PREVIEW_RELEASE_VERSION == 0)
```
instead?